### PR TITLE
Erto 21 recommendation box

### PR DESCRIPTION
**Pull request by xendk**

### CODE REVIEW

File: sites/all/modules/paragraphs/reol_rec_material_paragraph/reol_rec_material_paragraph.features.field_instance.inc [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+function reol_rec_material_paragraph_field_default_field_instances() {
+  $field_instances = array();
+
+  // Exported field_instance:
+  // 'paragraphs_item-recommended_material-field_rec_material'.
+  $field_instances['paragraphs_item-recommended_material-field_rec_material'] = array(
+    'bundle' => 'recommended_material',
+    'default_value' => NULL,
+    'deleted' => 0,
+    'description' => 'Kopier URLen på det materiale du vil referere til.',
```
Comment (tripox) [inline sites/all/modules/paragraphs/reol_rec_material_paragraph/reol_rec_material_paragraph.features.field_instance.inc]: Descriptions and labels should be wrapped in t() and translated.

File: sites/all/modules/paragraphs/reol_rec_material_paragraph/reol_rec_material_paragraph.features.inc [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ * reol_rec_material_paragraph.features.inc
+ */
+
+/**
+ * Implements hook_paragraphs_info().
+ */
+function reol_rec_material_paragraph_paragraphs_info() {
+  $items = array(
+    'recommended_material' => array(
+      'name' => 'Redaktøren anbefaler',
```
Comment (tripox) [inline sites/all/modules/paragraphs/reol_rec_material_paragraph/reol_rec_material_paragraph.features.inc]: t()?

File: sites/all/modules/paragraphs/reol_rec_material_paragraph/reol_rec_material_paragraph.features.field_instance.inc (hunk 1 above)
Comment (xendk) [reply sites/all/modules/paragraphs/reol_rec_material_paragraph/reol_rec_material_paragraph.features.field_instance.inc]: Tell that to features.

Field labels are not translatable without the i18n_field module, so we're living with danish strings.

File: sites/all/modules/paragraphs/reol_rec_material_paragraph/reol_rec_material_paragraph.features.inc (hunk 2 above)
Comment (xendk) [reply sites/all/modules/paragraphs/reol_rec_material_paragraph/reol_rec_material_paragraph.features.inc]: Again, features generated file.

**Pull request merged**